frontend/packages/console-dynamic-plugin-sdk/docs/console-extensions.md

```diff
@@ -972,6 +973,22 @@ Adds a new storage class provisioner as an option during storage class creation.
 
 ---
 
+## `console.tab`
+
+### Summary 
+
+Contributes a tab into an HorizontalNav component matching the sectionId.
```

Comment:
```suggestion
Adds a tab to an `HorizontalNav` matching the `contextId`.
```

Reply:
A code example would be good here.

Reply:
We should discuss how ordering works.

Reply:
`a HorizontalNav`

Reply:
I changed and fixed the text, unfortunately when adding TSDoc `@example` the docs are not generated correctly. I checked the generation but it seems to be a task big enough for a different PR.